meson/libsystemd/meson.build

```diff
@@ -1,9 +1,9 @@
-opt_systemd = get_option('systemd')
+opt_systemd = get_option('systemd-service')
 
-dep_systemd = dependency('libsystemd', required: opt_systemd)
-conf.set('HAVE_SYSTEMD', dep_systemd.found(), description: 'libsystemd')
-summary('libsystemd', dep_systemd.found(), bool_yn: true, section: 'Configuration')
+dep_libsystemd = dependency('libsystemd', required: opt_systemd)
```

Comment:
Should we still look for `libsystemd` when the `systemd-service` feature is disabled? Unless I'm mistaken the current code would still look for the library and set `HAVE_SYSTEMD` if it is found.